assets/dist/styles.css

```diff
@@ -0,0 +1,340 @@
+    font-weight: 100;
+    margin: 5px auto;
+}
+
+.link-intro {
+    margin: 10px 0 1em 0;
+    color: var(--paragraph-color);
+    letter-spacing: 1px;
+    max-width: 480px;
+}
+
+.astarte-tutorial__description {
+    margin: auto 2em;
+    color: var(--paragraph-color);
+    font-size: 1.1em;
+    max-width: 54em;
+    display: inline-block;
+    letter-spacing: 1px;
+}
+
+.tutorial-question-container {
+    flex: 1 0 41%;
+    margin: 5px;
+    min-height: 100px;
+    text-align: left;
+    padding-left: 1em;
+    min-width: 300px;
+}
+
+.tutorial-question-title {
+    color: var(--title-color);
+    font-weight: 700;
+    font-size: 1.2em;
+    color: var(--link-color);
+}
+
+.tutorial-question-answer {
+    color: var(--paragraph-color);
+    letter-spacing: 1px;
+    max-width: 450px;
+}
+
+.question-link:visited,
+.question-link:link {
+    color: var(--link-color);
+}
+
+.astarte-grid {
+    margin: 3em auto;
+    max-width: 75vw;
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.faq-question-container {
+    flex: 1 0 41%;
+    margin: 5px;
+    height: 100px;
+    text-align: left;
+    padding-left: 1em;
+    min-width: 300px;
+}
+
+.question-title {
+    color: var(--title-color);
+    font-weight: 100;
+    font-size: 1.2em;
+}
+
+.question-answer {
+    color: var(--paragraph-color);
+    letter-spacing: 1px;
+}
+
+.link,
+.link:link,
+.link:visited {
+    color: var(--link-color)
+}
+
+.sdk-intro-container {
+    width: 60vw;
+    margin: auto;
+}
+
+.sdk-content {
+    text-align: left;
+}
+
+.sdk-heading {
+    color: #686666;
+    font-size: 24px;
+    margin-bottom: 5px;
+}
+
+.sdk-paragraph {
+    margin: 10px auto;
+    color: #686666cc;
+    font-size: 1.1em;
+    max-width: 70vw;
+    display: inline-block;
+    letter-spacing: 1px;
+}
+
+.sdk-container {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+}
+
+.nav-container {
+    margin: 1em auto 0;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    flex-wrap: wrap;
+    width: 60vw;
+    padding: .5em 1em 0;
+    border: 1px solid var(--paragraph-color);
+    border-radius: 10px 10px 0 0;
+}
+
+.sdk-description {
+    margin: 0 auto 1em;
+    border: 1px solid var(--paragraph-color);
+    border-top: none;
+    border-radius: 0 0 10px 10px;
+    width: 60vw;
+    padding: 1em;
+
+}
+
+.underline:hover {
+    border-bottom: 3px solid var(--paragraph-color);
+
+}
+
+.sdk-item {
+    height: 2em;
+    padding: 5px 5px 10px;
+}
+
+.active {
+    color: var(--link-color);
+    border-bottom: 3px solid var(--link-color);
+}
+
+.code-container {
+    border: 1px solid var(--paragraph-color);
+    border-radius: 5px;
+    padding: 10px;
+    color: var(--title-color);
+    background-color: #68666615;
+    overflow: auto;
+}
+
+.code-inline {
+    border: 1px solid var(--paragraph-color);
+    border-radius: 5px;
+    color: var(--title-color);
+    background-color: #68666615;
+    text-decoration: none;
+}
```

Comment:
Missing new-line character.
It's usually a good practice to end files with a newline character

Reply:
Added newline character.